cryptoapis/src/endpoint/stats.ts

```diff
@@ -1,24 +1,32 @@
 import { Requester, Validator } from '@chainlink/external-adapter'
 import { AdapterRequest, Config } from '@chainlink/types'
+import { DEFAULT_ENDPOINT } from '../config'
 
-export const Name = 'difficulty'
+export const Name = 'stats'
```

Comment:
I was thinking of adhering to a certain name for this category of calls. Like we have `balance` and `price`, which also do not correspond to the actual endpoint naming.

Reply:
Well just because we have some code implemented currently does not make it the correct way to do it :)

I suggest you rename this for now, and we'll make sure to improve/refactor other adapter implementations in the future.

"balance", "price", "difficulty" & "height" are all data points for which one can query our system. There is no "stats" data point currently. Here you are mapping user request for "height" data point to a cryptoapis "bc-info" endpoint request.

Reply:
Changed it to `bc_info` for now. 

I think we have to find a uniform convention for these adapters that are returning blockchain info, while keeping `difficulty` (and now `height`) in backwards compatibility(probably by just delegating to `stats`/`bc_info` types when seeing these types of `endpoint`).